File: ProblemSolving/10주차/김재훈/BOJ_8958.py
# OX 퀴즈 문제

N = int(input())
for _ in range(N):
    S = input()
    total_score = 0
    score = 0
    for i in S:
        if i == 'X':
            score = 0
        else:
            score += 1
        total_score += score

    print(total_score)

File: ProblemSolving/10주차/김재훈/BOJ_8958.cpp
#include <iostream>

int main()
{
	int N;
	
	std::cin >> N;
	
	for (int i=0; i<N; i++)
	{
		std::string S;
		std::cin >> S;
		int total_score = 0;
		int score = 0;
		
		for (int j=0; j<S.size(); j++)
		{
			if (S[j] == 'X')
			{
				score = 0;
			}
			else
			{
				score += 1;
			}
			total_score += score;
		}
		std::cout << total_score << std::endl;
	}
}